src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -434,10 +434,10 @@ static void uct_ib_mlx5_devx_mr_lru_cleanup(uct_ib_mlx5_md_t *md)
 }
 
 static UCS_F_ALWAYS_INLINE int
-uct_ib_mlx5_devx_md_is_ksm_atomic_supported(uct_ib_mlx5_md_t *md)
+uct_ib_mlx5_devx_md_flags_test(uct_ib_mlx5_md_t *md, uint32_t flags)
 {
-    return ucs_test_all_flags(md->flags, UCT_IB_MLX5_MD_FLAG_KSM |
-                                         UCT_IB_MLX5_MD_FLAG_INDIRECT_ATOMICS);
+    UCS_STATIC_ASSERT(sizeof(md->flags) == sizeof(flags));
```

Comment:
This is basically a one-liner.
If we think it justifies a separate func, WDYT of sharing it between similar places which test MD flags, e.g in `dc_ml5x.c`, `rc_mlx5_iface.c`, etc? There's nothing particularly "devx" about it.

Reply:
Let's see how it goes in CI then refactor, there are multiple options:
 - delete one-liner at all
 - reuse it everywhere to test `UCT_IB_MLX5_MD_FLAG_XXX`  flags
 - rename it to something related to KSM/INDIRECT_ATOMICS or wider set of capabilities for traces and/or debugging purposes on control path

Reply:
Initially I defined `flags` with wrong type. The idea was to avoid a bug with overflow since even inline function does typecast. Most likely `md->flags` type will be changed with adding new flags but it's easy to miss usage.
IMO it's minor but good to have.

Reply:
IMO this function should be removed